packages/clerk-js/src/ui/components/UserProfile/__tests__/EnterpriseAccountsSection.test.tsx

```diff
@@ -0,0 +1,212 @@
+  });
+});
+
+const withOAuthBuiltInEnterpriseConnection = createFixtures.config(f => {
+  f.withUser({
+    enterprise_accounts: [
+      {
+        object: 'enterprise_account',
+        active: true,
+        first_name: 'Laura',
+        last_name: 'Serafim',
+        protocol: 'oauth',
+        provider_user_id: null,
+        public_metadata: {},
+        email_address: 'test@clerk.com',
+        provider: 'google',
+        enterprise_connection: {
+          object: 'enterprise_connection',
+          provider: 'google',
+          name: 'FooCorp',
+          id: 'ent_123',
+          active: true,
+          allow_idp_initiated: false,
+          allow_subdomains: false,
+          disable_additional_identifications: false,
+          sync_user_attributes: false,
+          domain: 'foocorp.com',
+          created_at: 123,
+          updated_at: 123,
+          logo_public_url: null,
+          protocol: 'oauth',
+        },
+        verification: {
+          status: 'verified',
+          strategy: 'oauth_google',
+          verified_at_client: 'foo',
+          attempts: 0,
+          error: {
+            code: 'identifier_already_signed_in',
+            long_message: "You're already signed in",
+            message: "You're already signed in",
+          },
+          expire_at: 123,
+          id: 'ver_123',
+          object: 'verification',
+        },
+        id: 'eac_123',
+      },
+    ],
+  });
+});
+
+const withOAuthCustomEnterpriseConnection = createFixtures.config(f => {
+  f.withUser({
+    enterprise_accounts: [
+      {
+        object: 'enterprise_account',
+        active: true,
+        first_name: 'Laura',
+        last_name: 'Serafim',
+        protocol: 'oauth',
+        provider_user_id: null,
+        public_metadata: {},
+        email_address: 'test@clerk.com',
+        provider: 'oauth_custom_roblox',
+        enterprise_connection: {
+          object: 'enterprise_connection',
+          provider: 'oauth_custom_roblox',
+          name: 'Roblox',
+          id: 'ent_123',
+          active: true,
+          allow_idp_initiated: false,
+          allow_subdomains: false,
+          disable_additional_identifications: false,
+          sync_user_attributes: false,
+          domain: 'foocorp.com',
+          created_at: 123,
+          updated_at: 123,
+          logo_public_url: null,
+          protocol: 'oauth',
+        },
+        verification: {
+          status: 'verified',
+          strategy: 'oauth_custom_roblox',
+          verified_at_client: 'foo',
+          attempts: 0,
+          error: {
+            code: 'identifier_already_signed_in',
+            long_message: "You're already signed in",
+            message: "You're already signed in",
+          },
+          expire_at: 123,
+          id: 'ver_123',
+          object: 'verification',
+        },
+        id: 'eac_123',
+      },
+    ],
+  });
+});
+
+const withSamlEnterpriseConnection = createFixtures.config(f => {
```

Comment:
Should also add a test for when `userSettings.saml` is disabled, or when the enterprise connection is inactive.